File: javascript/src/models/shopifyConfig.ts
// this file is @generated

export interface ShopifyConfig {
  secret: string;
}

export const ShopifyConfigSerializer = {
  _fromJsonObject(object: any): ShopifyConfig {
    return {
      secret: object["secret"],
    };
  },

  _toJsonObject(self: ShopifyConfig): any {
    return {
      secret: self.secret,
    };
  },
};

File: rust/src/models/shopify_config.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct ShopifyConfig {
    pub secret: String,
}

impl ShopifyConfig {
    pub fn new(secret: String) -> Self {
        Self { secret }
    }
}
